tutorials/plot_artifacts_correction_ssp.py

```diff
@@ -2,6 +2,12 @@
 Artifact Correction with SSP
 ============================
 
+This tutorial explains how to estimate Signal Subspace Projectors (SSP)
+for correction of ECG and EOG artifacts.
+
+See :ref:`<sphx_glr_auto_examples_io_plot_read_proj.py>` for how to read
```

Comment:
Still incorrect
```
/home/circleci/project/doc/auto_tutorials/plot_artifacts_correction_ssp.rst:16: WARNING: undefined label: <sphx_glr_auto_examples_io_plot_read_proj.py> (if the link has no caption the label must precede a section header)
```
Get rid of the `<>` and it will probably work, but locally you can do `make clean; make html_dev-noplot` and look at the last ~100 lines to see if the link is correct or not.